test/sds/SDS_Regression.ipynb

```diff
@@ -283,6 +283,10 @@
     "    else:\n",
     "        variable_slice = masked_variable\n",
     "\n",
+    "    # Some data are lon, lat rather than lat, lon (Python is row, column)\n",
```

Comment:
GPM/IMERGHH has dimensions (time, lon, lat) in that order. So the plotting routine was choking expecting (time, lat, lon).